services/registration/src/models/branch.ts

```diff
@@ -49,6 +49,11 @@ export interface Branch extends mongoose.Document {
     enabled?: boolean;
     group?: GradingGroupType;
   };
+  postSubmitEmailTemplate: {
+    enabled?: boolean;
+    subject: string;
+    content: string;
```

Comment:
enabled should always be present, so it should be `enabled: boolean`, and the other two can be undefined i think.